### ISSUE TITLE
DND Bypass

### ISSUE THREAD

**Issue by Digital39999** [Vencord/plugin-requests#131]
### Plugin Description

When you are dnd but you still get notifications from selected dms and servers.

### Existing Plugin for other mod

https://github.com/localip/powercord-dnd-bypass

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Inbestigator** [comment]
[I have submitted a merge request for this plugin here](https://github.com/Vendicated/Vencord/pull/2239)

**Comment by MahmoudAli96505** [comment]
any news about this plugin?

**Comment by Digital39999** [comment]
> any news about this plugin?

Yes, read the pull request.. https://github.com/Vendicated/Vencord/pull/2239#issuecomment-1995971931

**Comment by MahmoudAli96505** [comment]
> > any news about this plugin?
> 
> Yes, read the pull request.. [Vendicated/Vencord#2239 (comment)](https://github.com/Vendicated/Vencord/pull/2239#issuecomment-1995971931)

still didnt answer you?

**Comment by MahmoudAli96505** [comment]
Hello?

**Comment by nagaisa** [comment]
any update on the progress of this? or any idea when it may acatuly be added?

**Comment by Inbestigator** [comment]
> any update on the progress of this? or any idea when it may acatuly be added?

It’s completed and ready to merge, Vee probably just hasn’t gotten back to it in a while
